src/gui-commands.ts

```diff
@@ -24,6 +24,16 @@ function isNumber(n: string) {
   return !isNaN(parsedN) && isFinite(parsedN);
 }
 
+function validPassword(password: string) {
+  return (
+    password.length >= 8 &&
+    /[A-Z]/.test(password) &&
+    /[a-z]/.test(password) &&
+    /[0-9]/.test(password) &&
+    /[!@#$%^&*()_+*$]/.test(password)
```

Comment:
Where did this list come from?
These are also usually accepted as special characters for passwords: `{ } [ ] | \ : ; " ' < > , . ? /`
There is no way for the user to know what is considered a special character without looking into this code. Maybe add the list it to the command description?

Reply:
if we change the list now, we'll have to change it in 3 places, gui, cli, installer

Reply:
I think this list is fine as is since we are displaying them to the user.